StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/Link/LinkURLGeneratorTests.swift

```diff
@@ -39,17 +41,6 @@ class LinkURLGeneratorTests: XCTestCase {
         // Just make sure it doesn't fail
     }
 
-    func testURLCreationHorribleUnicode() {
-        var params = testParams
-        params.customerInfo.email = String(bytes: [0xD8, 0x00] as [UInt8], encoding: .utf16BigEndian)! // Unpaired UTF-16 surrogates
-        do {
-            _ = try LinkURLGenerator.url(params: params)
-            XCTFail("Encoding should fail for invalid data")
-        } catch {
-            XCTAssertTrue(error is EncodingError)
-        }
-    }
-
```

Comment:
We support horrible unicode now?

Reply:
As of Xcode 15.3, but not Xcode 15.1. We could have two tests, but this test isn't really necessary IMO